example/README.md

```diff
@@ -0,0 +1,22 @@
+## Installation
+```sh
+dart pub global activate flutterflow_cli
+```
+
+## How to use
+
+Base command with required flags:
+```sh
+`flutterflow export-code --project ProjectID --token APIToken`
```

Comment:
I think the inner backticks should be removed, currently it looks like this:
![Screenshot 2023-11-30 at 4 50 39 PM](https://github.com/FlutterFlow/flutterflow-cli/assets/92294640/755fc42a-58d6-4156-b392-2cb731daa7a8)